.github/workflows/publishing.yml

```diff
@@ -158,6 +160,54 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.REPO_BOT_ACCESS_TOKEN }}
 
+      - name: Retrieve wheel assets
+        run: |
+          assets_folder=/tmp/wheel_assets && mkdir "$assets_folder" && cd "$assets_folder"
+          
+          function download_assets {
+            mkdir "$1" && cd "$1"
+            release_id=`echo "$2" | jq -r ".$1.release_id"`
+            echo "Download $release_id assets..."
+            gh release download $release_id -R ${{ github.repository }}
```

Comment:
Even though the release is already created, the assets may not yet be uploaded to it. You hence need to wait for them:
```
          while [ -z `ls | egrep '...'` ]; do # where ... needs to be replaced with a check for each asset that you need.
            (gh release download $release_id -R ${{ github.repository }}) || true
            sleep 30 # waiting for assets to be created...
          done
```

Reply:
Fixed. For some reason, `ls | egrep ... ` could result in infinite loops when I tried it here. I used `! [ -f "$1" ]` condition to look for the corresponding `build_info` file instead.

Reply:
That should be fine. FYI, egrep produces an error if it doesn't match, so you usually should be able to do something like 
```
echo "$release_title" | (egrep -o "([0-9]{1,}\.)+[0-9]{1,}([A-Za-z0-9_\-\.]*)" || true
```